### PR TITLE
re-add search page

### PR DESCRIPTION
**Pull request by wslack**

### CODE REVIEW

**Comment by wslack** [comment]
aha, the nav moved. Got it.

File: pages/using-federalist/search.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: Search on Federalist Sites
+parent: Using Federalist
```
Comment (line47) [inline pages/using-federalist/search.md]: add permalink here in place of `parent`
`permalink: /pages/using-federalist/search/`

Comment (wslack) [reply pages/using-federalist/search.md]: Oh, the other way doesn't work now?

Comment (line47) [reply pages/using-federalist/search.md]: It will still work, just wanting it to be consistent

Comment (wslack) [reply pages/using-federalist/search.md]: Oh - it didn't work in the preview.

File: pages/using-federalist/search.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+title: Search on Federalist Sites
+permalink: /pages/using-federalist/search/
+---
+## Search on Federalist Sites
+
+It's easy to add search functionality to a static site.
+
+We recommend using [Search.gov][], a free site search and search analytics service for federal web sites. You will need to [register](https://search.usa.gov/signup) for Search.gov and follow their [instructions](https://search.gov/blog/go-live.html) to integrate this service with your Federalist site. For full details, visit [Search.gov][]. 
+
+If you'd prefer another solution, you can configure a tool like [lunrjs](https://lunrjs.com/) that creates a search function run using the client browser. An example of this is at the [18F blog](https://18f.gsa.gov/blog/). This avoids any dependancy on another service, but the search results are not as robust.
```
Comment (jseppi) [inline pages/using-federalist/search.md]: `dependancy` -> `dependency`

**Pull request merged**